File: output/ChatGPT/codenet/C++/Python/s754383079.py
a = input()
n = len(a)
b = input()
m = len(b)
ans = float('inf')

for i in range(1, n - m + 2):
    sum = 0
    for j in range(1, m + 1):
        if a[i + j - 1] != b[j]:
            sum += 1
    ans = min(ans, sum)

print(ans)

File: output/ChatGPT/codenet/C++/Java/s754383079.java
import java.util.Scanner;

public class s754383079 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String a = scanner.next();
        String b = scanner.next();
        int n = a.length();
        int m = b.length();
        int ans = Integer.MAX_VALUE;

        for (int i = 0; i + m - 1 < n; i++) {
            int sum = 0;
            for (int j = 0; j < m; j++) {
                if (a.charAt(i + j) != b.charAt(j)) {
                    sum++;
                }
            }
            ans = Math.min(ans, sum);
        }
        System.out.println(ans);
    }
}